exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/Database.kt

```diff
@@ -180,6 +183,7 @@ class Database private constructor(
          * but instead provides the details necessary to do so whenever a connection is required by a transaction.
          *
          * @param datasource The [DataSource] object to be used as a means of getting a connection.
+         * @param connectionAutoRegistration The connection provider for database. If not provided, will use ServiceLoaded [connectionInstanceImpl]
```

Comment:
`connectionInstanceImpl` is a private property that may not mean anything to users.
Please change this (and the other KDocs below) to something more general, like:
```
* @param connectionAutoRegistration The connection provider for database. If not provided,
* a service loader will be used to locate and load a provider for [DatabaseConnectionAutoRegistration].
```

Reply:
Fixed 🫡